dotnet-efcore/VirtualKeywordInEFCore/VirtualKeywordInEFCore/Models/Author_lazy.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace VirtualKeywordInEFCore.Models
+{
+    public class Author_lazy
```

Comment:
This should be renamed using pascal case instead of an underscore. Same goes for the `Book_lazy` class, variables like `authors_lazy`, elements in the test cases like `colleenHOOVER_author`, etc.

```suggestion
    public class AuthorLazy
```